core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/DiagnosticLog.java

```diff
@@ -172,62 +180,14 @@ public static class DiagnosticLogBuilder {
         private final String componentId;
         private Map<String, Object> input;
         private Map<String, Object> configurations;
-        private LogLevel logLevel;
+        private LogDetailLevel logDetailLevel;
 
         public DiagnosticLogBuilder(String componentId, String actionId) {
 
             this.componentId = componentId;
             this.actionId = actionId;
         }
 
-        /**
-         * Sets the log ID of the DiagnosticLog.
-         *
-         * @param logId the log ID to be set.
-         * @return the current DiagnosticLogBuilder instance.
-         */
-        public DiagnosticLogBuilder logId(String logId) {
-
-            this.logId = logId;
-            return this;
-        }
-
-        /**
-         * Sets the recordedAt timestamp of the DiagnosticLog.
-         *
-         * @param recordedAt the timestamp to be set.
-         * @return the current DiagnosticLogBuilder instance.
-         */
-        public DiagnosticLogBuilder recordedAt(Instant recordedAt) {
-
-            this.recordedAt = recordedAt;
-            return this;
-        }
-
-        /**
-         * Sets the request ID of the DiagnosticLog.
-         *
-         * @param requestId the request ID to be set.
-         * @return the current DiagnosticLogBuilder instance.
-         */
-        public DiagnosticLogBuilder requestId(String requestId) {
-
-            this.requestId = requestId;
-            return this;
-        }
```

Comment:
These public (All removed public methods and attributed in this PR) methods were removed since these have been introduced to the DiagnosticLog class in the previous kernel release. Since it was released recently, it would be safer to remove these public methods and attributes.